### ISSUE TITLE
Proposal: Improve tfchain wallet connector ux

### ISSUE THREAD

**Issue by MohamedElmdary** [threefoldtech/tfgrid-sdk-ts#506]
Based on what described in this issue https://github.com/threefoldtech/tfgrid-sdk-ts/issues/488 we are going to allow the user to have only one wallet.
This is kind of poor user experience and development experience, let's assume user have 3 *testnet* wallets - 2 *mainnet* wallet; also developers might have 2 dev, 2 qa, 1 test and 1 main.

Imagine if u have a list of your wallets whenever you login
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/31689104/3bf982d2-babb-4b2a-acd2-902b3fe7604f)

Clicking any one of predefined wallets should show a dialog asking the user to enter it's password
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/31689104/555ee6a5-e7b7-4a99-93af-078d02c667c2)

In case the user has no defined wallets he will need to add a new wallet
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/31689104/8c7ca8be-d9c1-46db-a771-9016f3e90a20)

### How this structure can be stored in localstorage?
basically we can use a flat pattern by prefixing passwordHash as follow
```yaml
localStorage:
  - wallet.[wallet name].passwordHash: encrypted mnemonic

  - wallet.devenet wallet1.hash_password_1: xxxxxxxxxxxxxxx1
  - wallet.devenet wallet2.hash_password_2: xxxxxxxxxxxxxxx2
```

### How to list wallets from localstorage?
```ts
for (let i = 0; i < localStorage.length; i++) {
    const key = localStorage.key(i)
    // Check if the key is following out structure
    // if so we should list it
}
```
